test/built-ins/Temporal/PlainDateTime/prototype/with/misspelled-property.js

```diff
@@ -0,0 +1,23 @@
+// Copyright (C) 2022 Igalia, S.L. All rights reserved.
+// This code is governed by the BSD license found in the LICENSE file.
+
+/*---
+description: Plural forms of known units are not acceptable
+esid: sec-temporal.plaindatetime.prototype.with
+features: [Temporal]
+---*/
+
+const instance = new Temporal.PlainDateTime(2000, 5, 2, 12, 34, 56, 987, 654, 321);
+
+const units = ["year", "month", "day", "hour", "minute", "second", "millisecond", "microsecond", "nanosecond"];
+
+units.forEach((unit) => {
+  let plural = `${unit}s`;
+  let options = {};
+  options[plural] = 1;
+  assert.throws(
+    TypeError,
+    () => instance.with(options),
+    `plural unit ("${plural}" vs "${unit}") is not acceptable`
```

Comment:
You could consider combining this with the previous empty-object test under the banner "at least one correctly-spelled property must be present". (and then also test a nonsense property such as `"other string"`)

Reply:
done